example/config/crd/bases/helm.sdk.operatorframework.io_nginxes.yaml

```diff
@@ -16,31 +13,12 @@ spec:
     singular: nginx
   scope: Namespaced
   versions:
-  - name: v1
```

Comment:
the tool generated `v1beta1` and it was manually updated to `v1` but not properly. 
So, changing it for the same layout of `v1` gen via SDK for we are able to use the example to test the helm-operator